.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
The reason I added them all is because the min and max ranges for Django are not fully overlapping.

Django 2.2: python 3.5-3.9
Django 3.2: python 3.6-3.10
Django 4.0: python 3.8-3.10

Cutting things off at python 3.7 since that's the lowest we now support:

Django 2.2: python 3.7-3.9 ← 3.7 for min, 3.9 for max
Django 3.2: python 3.7-3.10 ← 3.7 for min, 3.10 for max
Django 4.0: python 3.8-3.10 ← 3.8 for min, 3.10 for max

So to test all these version of Django at their own min and max versions starting from 3.7, we need 3.7, 3.8, 3.9, 3.10.